.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Cheff application build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main, dev ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
```

Comment:
Just wondering, why temurin?
To be clear it is a good choice. I just want to understand what was the decision driver.